src/main/java/com/desiato/puresynth/services/AudioService.java

```diff
@@ -14,7 +14,10 @@ public class AudioService {
     private static final int BITS_DEPTH = 16;
     private static final boolean BIG_ENDIAN = false;
 
-    public byte[] generateSineWaveFile(double frequency, double durationInSeconds) throws IOException, LineUnavailableException {
+    public byte[] generateSineWaveFile(double frequency,
```

Comment:
Same with indention. I think it would be good to add a rule to protect against this type of indention.

Reply:
I have looked into checkstyle, I found `ParameterNumber` module, but it specify the total number of parameters per method. I have to look more into that to see if an option for specifying parameters' indentation exists.

Reply:
I think Google Checkstyle config had something like that, though I'm not sure. Maybe it could also be a configration in the IDE.